lib/publisher.ex

```diff
@@ -336,6 +341,11 @@ defmodule GenRMQ.Publisher do
     {:ok, %{channel: channel, module: module, config: config, conn: conn}}
   end
 
+  defp parse_config(config) do
+    config
+    |> Keyword.put(:connection, Keyword.get(config, :connection, config[:uri]))
```

Comment:
nit: Let's not use pipe (`|>`) here, I think it's clearer without it when only one function is involved.

Reply:
Made this change.